bindings/java/src/main/java/org/ucx/jucx/examples/UcxBenchmark.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) Mellanox Technologies Ltd. 2019. ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+package org.ucx.jucx.examples;
+
+import org.ucx.jucx.ucp.UcpContext;
+import org.ucx.jucx.ucp.UcpParams;
+import org.ucx.jucx.ucp.UcpWorker;
+import org.ucx.jucx.ucp.UcpWorkerParams;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Stack;
+
+public abstract class UcxBenchmark {
+
+    private static long BYTES_IN_GIGABIT = 125_000_000L;
+
+    protected static Map<String, String> argsMap = new HashMap<>();
+
+    // Stack of closable resources (context, worker, etc.) to be closed at the end.
+    protected static Stack<Closeable> resources = new Stack<>();
+
+    protected static UcpContext context;
+
+    protected static UcpWorker worker;
+
+    protected static int serverPort;
+
+    protected static int numIterations;
+
+    protected static int totalSize;
+
+    private static String DESCRIPTION = "JUCX benchmark.\n" +
+        "Run: java -cp jucx.jar org.ucx.jucx.examples.BENCHMARK_CLASS [parameter=value]\n\n" +
+        "Parameters:\n" +
+        "h - print help\n" +
```

Comment:
looks like a nonstandard to pass parameters
i would expect -h, -s, etc. not s=...

Reply:
Java doesn't have in the standart library any getops variant. There are good external libraries though (https://picocli.info/) . To not include any external libraries and not complicate code with argument parsing state machine, the easiest way is just to pass parameters as key value pairs and split based on `=` sign.

Reply:
ok, so pls update help message to reflect the expected argument syntax